meta-xt-prod-cockpit-rcar-driver-domain/recipes-kernel/linux/files/r8a7795-h3ulcb-4x2g-kf-doma.dts

```diff
@@ -48,14 +48,25 @@
 	};
 
 	rvgc {
-		nr-displays = <0x1>;
-		display-mappings = <0x1>;
-		display-layer = <0x3>;
-
-		dma-ranges = <0x0 0x0 0x0 0x0 0x1 0x0>;
-		rvgc-memory {
-			memory-region = <&cr7_cma>;
-		};
+                dma-ranges = <0x0 0x0 0x0 0x0 0x1 0x0>;
+                rvgc-memory {
+                        memory-region = <&cr7_cma>;
+                };
+
+                displays {
+                        display-0 {
```

Comment:
we have 2 displays for android, but here is one.
I suppose this one belong to cluster, if so, than parameters must be the following
layer-map=<0x4>;
size-w=<800>;
size-h=<600>;
pos-x=<560>; 
pos-y=<60>;   

if not - one more display has to be added.

Reply:
@lorc   You absolutely right, it will be separate change